x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/job/JobManager.java

```diff
@@ -605,7 +605,7 @@ private void auditFilterChanges(String jobId, String filterId, Set<String> added
 
     private static void appendCommaSeparatedSet(Set<String> items, StringBuilder sb) {
         sb.append("[");
-        Strings.collectionToDelimitedString(items, ", ", "'", "'", sb);
+        Strings.collectionToDelimitedString(items, ", ", sb);
```

Comment:
Ah I missed this usage when I looked at your other PR - TBH I missed the whole method without the limit. Here there actually is a non-empty prefix and suffix... I'm inclined to think that it probably wouldn't have a huge effect if we don't wrap the items with single quotes, but technically we're changing something here. How do you feel about it?

Reply:
Oof that's a subtle one, the dataflow-to-here results pane in IntelliJ uses a proportional font in which "" and "'" are indistinguishable to my eyes.

Reply:
Ahh this is a great workaround!